libraries/AP_Scripting/examples/ahrs-source-gps-optflow.lua

```diff
@@ -112,10 +111,7 @@ function update()
 
   -- get opticalflow innovations from ahrs (only x and y values are valid)
   local opticalflow_over_threshold = true
-  local opticalflow_xy_innov = 0
-  local opticalflow_innov = Vector3f()
-  local opticalflow_var = Vector3f()
-  opticalflow_innov, opticalflow_var = ahrs:get_vel_innovations_and_variances_for_source(5)
+  local opticalflow_innov = ahrs:get_vel_innovations_and_variances_for_source(5)
   if (opticalflow_innov) then
     opticalflow_xy_innov = math.sqrt(opticalflow_innov:x() * opticalflow_innov:x() + opticalflow_innov:y() * opticalflow_innov:y())
```

Comment:
do we not need to put "local" in front of opticalflow_xy_innov?

Reply:
your right. I hope this is the sort of thing that the global checks should catch when we get around to enabling that.

Reply:
fixed